stdlib/public/SwiftShims/module.modulemap

```diff
@@ -19,6 +20,11 @@ module SwiftShims {
   export *
 }
 
+module SwiftShimsInline {
```

Comment:
Maybe this should be "_SwiftPlatformOverlayShims", to follow the convention below?